src/provider/common/Injectables.ts

```diff
@@ -11,7 +11,7 @@ enum Injectable {
     ENVIRONMENT,
     INTENT_HANDLER,
     MODEL,
-    SELECTOR,
+    RESOLVER,
```

Comment:
Any particular reason for this rename?

Reply:
Its no longer called selector

Reply:
So that's a no I guess

Reply:
Sorry let me be more specific.  The JIRA refers to the UI as the 'resolver'.  It was coded using this in mind.  Speaking to @pjbroadbent it sounds like selector was used previously as there an existing resolver component which is no more.  The selector was not a great name so I think it is a welcomed change.

Reply:
Fair enough. Thanks!